sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -240,6 +240,32 @@ class RocksDBSuite extends AlsoTestWithChangelogCheckpointingEnabled with Shared
     }
   }
 
+  testWithChangelogCheckpointingEnabled("Rollback multiple versions") {
```

Comment:
Can we give a more descriptive test name ? Also maybe add a comment linking the SPARK issue and just a small description of which case this test is validating